src/sensors/scripts/sensor.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
```

Comment:
More ROS Nav stuff that can be removed I think

Reply:
Yes